develop/tutorials/articles/193-adaptive-media/00-using-adaptive-media-apis-intro.markdown

```diff
@@ -0,0 +1,283 @@
+# Configuring Adaptive Media APIs
+
+The Adaptive Media APIs give you lots of control over your media. You can create
+unique attributes for your media types, search and obtain information about
+adaptive media such as the width and height of an image, and obtain URLs for an 
+adaptive media. This tutorial demonstrates these features.
+
+<!-- Are there any other things you can do with the API that we should mention? -->
+
+First, you must deploy the proper Adaptive Media OSGi modules. This is covered
+next.
+
+## Deploying Adaptive Media
+
+Development of Adaptive Media modules is done in the master branch of an
+independent repository of Liferay at the following URL:
+[https://github.com/liferay/com-liferay-adaptive-media](https://www.google.com/url?q=https://github.com/liferay/com-liferay-adaptive-media&sa=D&ust=1482880889869000&usg=AFQjCNHzWR-uBL7ROpGnHzG2agFqs89miQ)
+
+<!-- Shouldn't we just point to the Liferay-portal repo and consider that what
```

Comment:
Shouldn't we just point to the Liferay-portal repo and consider that what is in Portal is always stable? I know at the moment the repo isn't up to date in Portal.

I still mention the other repo if they want the bleeding edge changes.